src/libraries/Microsoft.Extensions.Caching.Abstractions/src/Microsoft.Extensions.Caching.Abstractions.csproj

```diff
@@ -1,22 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Description>Caching abstractions for in-memory cache and distributed cache.
-Commonly used types:
-Microsoft.Extensions.Caching.Distributed.IDistributedCache
-Microsoft.Extensions.Caching.Memory.IMemoryCache</Description>
-    <TargetFrameworks>netstandard2.0;$(DefaultNetCoreTargetFramework)</TargetFrameworks>
-    <TargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <PackageTags>cache;memorycache;distributedcache</PackageTags>
-    <IsPackable>true</IsPackable>
-    <IsShipping>true</IsShipping>
   </PropertyGroup>
 
   <ItemGroup>
+    <Compile Include="**/*.cs" />
```

Comment:
I may have missed this in past PRs.  In general we try to avoid the wildcards in the CSProj's but that's just because we condition imports.  There are other ways to do this that might be preferable.  Maybe we set `EnableDefaultItems`  @ViktorHofer any opinions on this one?

Reply:
Yes, what Eric said. If we don't plan to multi-target for different OSs then I would use `EnableDefaultItems` here.